src/System.Private.CoreLib/shared/System/Buffers/StandardFormat.cs

```diff
@@ -0,0 +1,174 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Purely a logistical comment, but should we instead do this by moving the code in corefx to shared, letting it mirror over, and then updating the rest of coreclr with the necessary changes?  I think that would make the history easier to follow.

Reply:
That sounds like a good idea.